packages/text-editor/src/components/extension/nodeHighlight.ts

```diff
@@ -29,7 +34,7 @@ export const NodeHighlightExtension = Extension.create<NodeHighlightExtensionOpt
 
             const range = getMarkRange(doc.resolve(pos), schema.marks[NodeUuidExtension.name])
 
-            if (range === null || range === undefined) {
+            if (!isRange(range)) {
```

Comment:
May be?
```suggestion
            if (range == null) {
```
will be enough?